engine/unity5/Assets/Scripts/Input/InputControl.cs

```diff
@@ -188,6 +188,11 @@ public static KeyMapping setKey(string name, int controlIndex, CustomInput prima
             return mPlayerList[controlIndex].setKey(name, argToInput(primary), null, isTankDrive);
         }
 
+        public static KeyMapping setKey(string name, int controlIndex, KeyCode primary, bool isTankDrive)
```

Comment:
Mack said to use pascal case naming conventions for this so 

`public static KeyMapping SetKey(string name, int controlIndex, KeyCode primary, bool isTankDrive)`

I'll just auto-merge this once that is fixed